archinstall/lib/disk/user_guides.py

```diff
@@ -36,7 +36,7 @@ def suggest_single_disk_layout(block_device, default_filesystem=None, advanced_o
 	layout[block_device.path]['partitions'].append({
 		# Root
 		"type" : "primary",
-		"start" : "518MB",
+		"start" : "512MiB",
```

Comment:
Archinstall uses [MB not MiB](https://github.com/archlinux/archinstall/issues/674#issuecomment-952837370) in all it's internal calculations, and `parted` supports it as well. So we uniform all numbers to MB. The code to do the bytes -> GB function was introduced [here](https://github.com/archlinux/archinstall/commit/6d0c55e5d610abd199c1dc0430576535dab894b7).

Reply:
Discussed on Discord, archinstall appears to use MiB/GiB for calculations instead of MB/GB. parted also uses separate conversion values for MiB/GiB and MB/GB https://www.gnu.org/software/parted/manual/parted.html#unit